_src/assets/js/main.js

```diff
@@ -61,87 +62,89 @@ let dataCard = {
   error: ""
 };
 
-// cheking if exist data on localStorage
-//Refactor: retrieve from LS data and assign it to let variable
+// cheking if exist data on localStorage /Refactor: retrieve from LS data and assign it to let variable
 let savedData = localStorage.getItem("data");
+
+// default checkout for palettes
+if (!savedData){
+  inputBlue.checked = true;
+  userCard.classList.add('color-blue');
+}
 //Refactor: if we have LS, parse it and fill Form
 if (savedData) {
   // to fill datacard
   dataCard = JSON.parse(savedData);
   fillSavedForm();
 }
 
-// fill form with data on localStorage
-//refactor: modify this loooong function to separate ones and invoke them as callbacks into a wider one. For  ifs, use ternary operators
-function fillSavedForm() {
-  //refactor: create function for fill name and job
-  if (dataCard.name) {
-    name.innerHTML = dataCard.name;
-  } else {
-    name.innerHTML = "Nombre Apellido";
-  }
+//functions to refactoring
+
+function fillName(){
+  dataCard.name? name.innerHTML = dataCard.name: name.innerHTML = "Nombre Apellido";
   inputName.value = dataCard.name;
-  //job
-  if (dataCard.job) {
-    job.innerHTML = dataCard.job;
-  } else {
-    job.innerHTML = "Front-end developer";
-  }
+}
+
+function fillJob(){
+  dataCard.job?job.innerHTML = dataCard.job:job.innerHTML = "Front-end developer";
   inputJob.value = dataCard.job;
+}
 
-  //-----second function
-  //icons
-  //third and fourth functions: one for icons href (include Linkedin icons[2].href ), and another one for phone, mail and github value
+function fillIcons(){
   icons[0].href = "tel: +34" + dataCard.phone;
   phone.value = dataCard.phone;
 
   icons[1].href = "mailto:" + dataCard.email;
   mail.value = dataCard.email;
 
-  icons[3].href = "https://github.com/" + dataCard.github;
-  github.value = dataCard.github;
-  // photo
-  //refactor: fifth function. Plus amend small preview image at reload
+  icons[2].href = "https://linkedin.com/" + dataCard.email;
+  mail.value = dataCard.email;
+
+  icons[3].href = "https://github.com/" + dataCard.linkedin;
+  linkedin.value = dataCard.linkedin;
+}
```

Comment:
From line 99 to 103, are the dataCard properties matching the values for each icon href? Confused here when I see that Linkedin matches dataCard.email: `icons[2].href = "https://linkedin.com/" + dataCard.email;`
Same goes for gitHuh at line 102, could you please help me understand it? Thanks a lot!

Reply:
Thanks for the warning. It's a mistake . It is already corrected. Now I think it is correct.

```
function fillIcons(){
  icons[0].href = "tel: +34" + dataCard.phone;
  phone.value = dataCard.phone;

  icons[1].href = "mailto:" + dataCard.email;
  mail.value = dataCard.email;

  icons[2].href = "https://linkedin.com/" + dataCard.linkedin;
  mail.value = dataCard.linkedin;

  icons[3].href = "https://github.com/" + dataCard.github;
  linkedin.value = dataCard.github;
}
```